bin/create-probot-app.js

```diff
@@ -16,7 +16,7 @@ const TEMPLATE_REPO_URL = 'https://github.com/probot/template.git'
 
 program
   .usage('[options] [destination]')
-  .option('-p, --pkgname <package-name>', 'Package name')
+  .option('-n, --appName <app-name>', 'What is the name of your app?')
```

Comment:
I think for consistency, this should be "The name of your app" or "App name." Do you mind making that change?

(I think I had suggested the more verbose version with a question mark, but looking at it now it doesn't feel write next to all the other ones.)

Reply:
No problem! Updated in https://github.com/probot/create-probot-app/pull/10/commits/24c689c3b93b9cd5319a5c56886681cfaf346764